src/databricks/labs/ucx/queries/assessment/main/00_7_count_storage_credentials.sql

```diff
@@ -0,0 +1,4 @@
+/* --title 'Total Storage Credentials' */
+SELECT
+  COUNT(*) AS count_total_storage_credentials
+FROM system.information_schema.storage_credentials
```

Comment:
okay, for storage credentials it's a bit more complicated:

1. do `if len(self._ws.storage_credentials.list()) > 200: raise RuntimeWarning('migration will breach UC limits')` in https://github.com/databrickslabs/ucx?tab=readme-ov-file#migrate-credentials-command
2. do this widget over `inventory.external_locations` table and highlight the value as red if it's over 500. Do this via widget override - https://github.com/databrickslabs/lsql/blob/main/docs/dashboards.md#overrides

Reply:
Atm we [create one access connector per storage account](https://github.com/databrickslabs/ucx/blob/5de46c0f43c5f5e33b74a4cfe2d628c0859eab2d/src/databricks/labs/ucx/azure/access.py#L313). [The storage accounts are derived from the external locations](https://github.com/databrickslabs/ucx/blob/5de46c0f43c5f5e33b74a4cfe2d628c0859eab2d/src/databricks/labs/ucx/azure/access.py#L367). You could warn in the dashboard when the (unique) storage accounts are above 200

Reply:
@nfx I and @JCZuurmond added the override json to highlight the value, but it did not take effect in the dashboard. We have narrowed it down to the CounterFieldEncoding dataclass https://github.com/databrickslabs/lsql/blob/main/src/databricks/labs/lsql/lakeview/model.py#L521 not having the 'style' nested field. Will you be able to generate this from the API or the fields have to be added manually?

e.g. json
`{"spec": {
  "version": 2,
  "widgetType": "counter",
  "encodings": {
    "value": {
      "fieldName": "count_total_storage_credentials",
      "rowNumber": 0,
      "style": {
        "rules": [
          {
            "condition": {
              "operator": ">",
              "operand": { "type": "data-value", "value": "20" }
            },
            "color": "#E92828"
          }
        ]
      },
      "displayName": "count_total_storage_credentials"
    }
  },
  "frame": {
    "description": "",
    "showDescription": false,
    "showTitle": true,
    "title": "Total Storage Credentials"
  }
}}`

Reply:
https://github.com/databrickslabs/lsql/issues/234